### PR TITLE
feat: OPX-3.2.0

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
* Update: Set SAI flag on vlan api
* Update: NAS/NDI does not need to be aware of the SAI Default group
* Bugfix: To achieve flexible trap management packet queue need
        to be supported at trap group level instead of at trap level

Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.69])
-AC_INIT([opx-nas-ndi], [4.8.0+opx2], [ops-dev@lists.openswitch.net])
+AC_INIT([opx-nas-ndi], [5.4.0], [ops-dev@lists.openswitch.net])
```
Comment (rakeshdatta) [inline configure.ac:21]: 5.4.0 from 4.8.0??

File: inc/opx/nas_ndi_router_interface_utl.h [hunk 2]
```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline inc/opx/nas_ndi_router_interface_utl.h]: change copyright plz

File: src/nas_ndi_router_interface_utl.cpp [hunk 3]
```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline src/nas_ndi_router_interface_utl.cpp]: copyright plz

File: src/unit_test/nas_ndi_ipmc_ut.cpp [hunk 4]
```diff
@@ -0,0 +1,687 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline src/unit_test/nas_ndi_ipmc_ut.cpp]: change copyright plz

**Review by rakeshdatta — CHANGES_REQUESTED** [review]

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 193739a075850ab030014e72b167b22981a0f4cb

**Review by rakeshdatta — APPROVED** [review]

**Pull request merged**